### ISSUE TITLE
gnt-instance rename: no attribute 'ignore_ip'

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#142]
Originally reported of Google Code with ID 86.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

# gnt-cluster --version
gnt-cluster (ganeti) 2.1.0~rc4

# gnt-cluster version
Software version: 2.1.0~rc4
Internode protocol: 20
Configuration format: 2010000
OS api version: 15
Export interface: 0

<b>What distribution are you using?</b>

Gentoo

<b>What steps will reproduce the problem?</b>
1. gnt-instance rename gvm04 gvm05
<b>2.</b>
<b>3.</b>

<b>What is the expected output? What do you see instead?</b>

I expected it to not show me an error :) This is what I got instead:

 # gnt-instance rename gvm04 gvm05
Traceback (most recent call last):
  File "/usr/sbin/gnt-instance", line 1425, in <module>
    override={"tag_type": constants.TAG_INSTANCE}))
  File "/usr/lib64/python2.5/site-packages/ganeti/cli.py", line 1373, in 
GenericMain
    result = func(options, args)
  File "/usr/sbin/gnt-instance", line 596, in RenameInstance
    ignore_ip=opts.ignore_ip)
AttributeError: Values instance has no attribute 'ignore_ip'

<b>Please provide any additional information below.</b>

I checked the ganeti logs and I don't see anything so it must not have 
gotten far enough to get to the daemons. I should note that I am using my 
own instance definition which you can find the source at [1] in case its 
related to that.

Thanks-

[1] http://git.osuosl.org/?p=ganeti-instance-image.git;a=summary

```
Originally added on 2010-02-09 01:15:45 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Ah, this is indeed a bug that was not caught by our QA suite since we don't check
gnt-instance rename functionality in it.

Thanks for letting us know.
iustin
```
Originally added on 2010-02-09 09:10:17 +0000 UTC.

Changed State: Accepted

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in git and will be in the next RC (or 2.1.0 final).

If you want, the fix is a one-line change, see
http://git.ganeti.org/ganeti.git/?a=commit;h=c4a73785d133c9ad329cff1653b23ba75a4e213d.
```
Originally added on 2010-02-09 09:35:15 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.1

**Comment by Ganeti-Issues-Migrator** [comment]
```
I can confirm this one-liner fixes it. Thanks for the quick turn around!
```
Originally added on 2010-02-09 17:58:11 +0000 UTC.
